File: Year 2/Sem2/Web/Lab7/book.php
<?php

class Book implements JsonSerializable {
	private $author;
	private $genre;
	private $pages;	
	private $title;

	public function __construct($author, $genre, $pages, $title, $lent) {
		$this->author = $author;
		$this->genre = $genre;
		$this->pages = $pages;
		$this->title = $title;
		$this->lent = $lent;
	}

	public function getAuthor() {
		return $this->author;
	}
	public function getGenre() {
		return $this->genre;
	}
	public function getPages() {
		return $this->pages;
	}
	public function getTitle() {
		return $this->title;
	}
	
	public function getLent() {
		return $this->lent;
	}

	public function jsonSerialize() {
        $vars = get_object_vars($this);
        return $vars;
    }
}

?>

File: Year 2/Sem2/Web/ExamPractice/ASP_Other/ASP_CRUD - Copy/ASPLogin - Copy/ASPLogin/Models/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ASPLogin.Models
{
    public class Book
    {
        public string Author { get; set; }
        public string Genre { get; set; }
        public int Lent { get; set; }
        public int Pages { get; set; }
        public string Title { get; set; }
    }
}